apps/api/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "api",
+  "version": "0.0.1",
+  "description": "",
+  "author": "",
+  "private": true,
+  "license": "UNLICENSED",
+  "scripts": {
```

Comment:
Add node version as 20
Check this
https://github.com/OsmosysSoftware/osmo-x/blob/399d75882f54c9e1dbb2f472d7c5d86feaa01227/apps/api/package.json#L7

Reply:
Updated